build-system/eslint-rules/html-template.js

```diff
@@ -181,7 +181,7 @@ function create(context) {
     const matches = [];
 
     let match;
-    while ((match = invalid.exec(string))) {
+    while ((match = invalid.exec(string)) && !string.includes('<svg')) {
```

Comment:
Why was this changed?

Reply:
I reverted the change and added svgs as exceptions (as they are foreign elements) to the invalid regex line a few lines above. If I don't make this change, the linter will recognize the svg-specific tags as invalid elements,
when they are actually valid.

https://html.spec.whatwg.org/multipage/syntax.html#foreign-elements